packages/core/src/internal/deployer.ts

```diff
@@ -74,10 +76,10 @@ export class Deployer {
     accounts: string[],
     defaultSender: string
   ): Promise<DeploymentResult> {
-    this._emitDeploymentStartEvent(ignitionModule.id, this._deploymentDir);
-
     let deploymentState = await this._getOrInitializeDeploymentState();
 
+    this._emitDeploymentStartEvent(ignitionModule.id, this._deploymentDir);
```

Comment:
This resolves the issue with the `Ignition starting...` text appearing before the chainId error.

There is a larger change that this suggests though, which is to rework our unexpected exception handling to clear the UI.

Reply:
I don't full understand this, but let's merge this pr, as it looks good as a small/local fix.